unison-src/transcripts-round-trip/main.output.md

```diff
@@ -34,15 +34,15 @@ x = 1 + 1
   most recent, along with the command that got us there. Try:
   
     `fork 2 .old`             
-    `fork #bt17giel42 .old`   to make an old namespace
+    `fork #m41m2ql36i .old`   to make an old namespace
```

Comment:
Do we expect changed hashes here?

Reply:
I don't know why this changes. The transcript is starting up with a different root hash than before for some reason.

Reply:
This happens in trunk as well, so it's unrelated to this change.

Reply:
Oh weird, ok.